src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentsState = { ...state };
```

Comment:
The variable name `currentsState` is a bit misleading as it suggests multiple states, which is not the case. Consider renaming it to `currentState` to accurately reflect that it represents a single state object at the current step in the actions array. [NAMING]